menu/cbs/menu_cbs_get_value.c

```diff
@@ -631,15 +631,22 @@ static void menu_action_setting_disp_set_label_input_desc_kbd(
    char *s2, size_t len2)
 {
    char desc[PATH_MAX_LENGTH];
-   unsigned key_id;
+   unsigned key_id, id;
    unsigned remap_id;
+   unsigned offset = 0;
+
    settings_t *settings = config_get_ptr();
 
    if (!settings)
       return;
 
+   offset = type / ((MENU_SETTINGS_INPUT_DESC_KBD_END - (MENU_SETTINGS_INPUT_DESC_KBD_END - MENU_SETTINGS_INPUT_DESC_KBD_BEGIN))) - 1;
+
+   id = (type / (offset + 1)) - MENU_SETTINGS_INPUT_DESC_KBD_BEGIN;
```

Comment:
btw I have no idea of what the math here means, it just worked so I settled with it